File: fix_page.js
const fs = require('fs');
let content = fs.readFileSync('src/app/page.tsx', 'utf8');

// 1. Add import
if (!content.includes('import ThemeSwitcher from')) {
  content = content.replace(
    /import LocationAutocomplete from '@\/components\/LocationAutocomplete';/,
    `import LocationAutocomplete from '@/components/LocationAutocomplete';\nimport ThemeSwitcher from '@/components/ThemeSwitcher';`
  );
}

fs.writeFileSync('src/app/page.tsx', content);
console.log('Fixed page import');

File: fix_page.py
import re

with open('src/app/page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix the broken error setting
pattern = r"setError\(SERVER HTTP ERROR \\: \\\);"
replacement = r"setError(SERVER HTTP ERROR : );"
content = re.sub(pattern, replacement, content)

with open('src/app/page.tsx', 'w', encoding='utf-8') as f:
    f.write(content)
